src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -67,16 +67,16 @@
 											<!-- iOS workaround #361 DoubleAnimationUsingKeyFrames having a small delay (0.5 seconds) before animating -->
 											<ios:DoubleAnimation Storyboard.TargetName="PressRing"
 																 Storyboard.TargetProperty="Opacity"
-																 Duration="0:0:0.15"
-																 EasingFunction="{StaticResource Material_EaseInOutFunction}"
+																 Duration="{StaticResource MaterialAnimationDuration}"
+																 EasingFunction="{StaticResource MaterialEaseInOutFunction}"
 																 From="0"
 																 To="0.25" />
```

Comment:
There a lot of place where 0.25 opacity were used. If we are reusing "this concept", let's try to put it in a resource?
Same for .38 and .7; Could we use and/or expand the ones in ColorPalette?

Reply:
There are those already to use. I'm tihnking any 0.38 should be MaterialLowOpacity and 0.7 should be MaterialMediumOpacity.


	<!-- Opacity for control state overlays -->
	<x:Double x:Key="MaterialMediumOpacity">0.74</x:Double>
	<x:Double x:Key="MaterialHoverOpacity">0.04</x:Double>
	<x:Double x:Key="MaterialLowOpacity">0.38</x:Double>
	<x:Double x:Key="MaterialFocusedOpacity">0.12</x:Double>
	<x:Double x:Key="MaterialPressedOpacity">0.10</x:Double>
	<x:Double x:Key="MaterialDraggedOpacity">0.08</x:Double>
	<x:Double x:Key="MaterialSelectedOpacity">0.08</x:Double>

Reply:
I did the 0.25 but 0.7 is only used for scaling the placeholder of the Textbox and PasswordBox